If-CI-checks-fail-on-your-PR.md

```diff
@@ -1,42 +1,71 @@
 
-1. If you see a warning that your PR has a **merge conflict**, you will need to resolve the conflict. To do this, follow the "merge from develop" instructions in  [step 5 of our PR guide](https://github.com/oppia/oppia/wiki/Rules-for-making-PRs#step-5-address-review-comments-until-all-reviewers-approve). Once you push the merge commit to your feature branch on GitHub, the merge conflict message will disappear.
+1. At the bottom of the PR thread, you will see a set of results that looks similar to the following. (**Note:** If your failure happens in the merge queue, you might need to click on "View details" to see the corresponding table.)
 
-2. If the issue isn't a merge conflict, then there are two possibilities: your code could be wrong, or the test/check on Oppia's develop branch could be incorrect. You will need to determine which of these cases it is. Note that the test/check on Oppia's develop branch could be incorrect for two reasons:
+     ![Screenshot of PR CI results](images/ghciSample.png)
 
-    * Sometimes, the tests in develop may be "flaky", which means they pass sometimes and fail sometimes, even though the code has not changed. The dev workflow team is trying to reduce cases like this, though they may sometimes still happen.
+   In the example above, the lint checks and Mypy checks have failed. Also, two checks were skipped due to the failure of one of the e2e tests. (This is because, in the event of an e2e test failure, any remaining e2e tests that are queued or currently running will be terminated and marked as "skipped.") The rest of the tests have passed.
 
-    * If an incorrect PR was recently merged to develop, this could cause the tests to fail on develop. If you see this happening, please follow the [[Revert and Regression Policy|Revert-and-Regression-Policy]] so that we can revert the problematic changes as soon as possible, since the develop branch should always be passing checks.
+1. Look for the specific test that is failing, and click the "Details" link next to the failing test to go to the GitHub Actions page for that test run.
+
+1. In GitHub Actions, select the job under 'Jobs' to see the logs for that particular job.
+
+1. Read the GitHub Actions log to figure out what the actual failure is.
+
+     ![Screenshot of GitHub Actions CI logs page](images/githubActionsLogs.png)
+
+1. Search for the error message on the [issue tracker](https://github.com/oppia/oppia/issues) to see whether the error has been reported before. If there's an open issue for the same error, see [What to do if the failure is due to an existing issue](#what-to-do-if-the-failure-is-due-to-an-existing-issue), below.
```

Comment:
What should the developer do when there are multiple issues in the pr? Do they open multiple issues or 1?

Reply:
They should open multiple issues. I have clarified this under "Look for the specific test ... " instruction above -- thanks!

Reply:
Maybe I missing something, but I don't see anything in the changes that state to open multiple issues if there is more than one?